backend/src/main/java/ca/bc/gov/hlth/hnweb/converter/hl7v3/FindCandidatesConverter.java

```diff
@@ -42,7 +43,11 @@ public FindCandidatesRequest convertRequest(NameSearchRequest nameSearchRequest)
 
 		findCandidatesRequest.setName(name);
 		findCandidatesRequest.setBirthDate(V3MessageUtil.dateOnlyFormatter.format(nameSearchRequest.getDateOfBirth()));
-		findCandidatesRequest.setGender(nameSearchRequest.getGender());
+		String gender = nameSearchRequest.getGender();
+		if (StringUtils.isNotEmpty(gender) && gender.equals("U")) {
+			findCandidatesRequest.setGender("UNK");
+		} else
+			findCandidatesRequest.setGender(nameSearchRequest.getGender());
```

Comment:
This code can be simplified:
- you can just use StringUtils.equals since it's null safe.
- Also you can just use a ternary expression to determine gender and then set it once on findCandidatesRequest.